packages/fuel-gauge/src/predicate/predicate-with-contract.test.ts

```diff
@@ -117,9 +115,7 @@ describe('Predicate', () => {
 
       await fundPredicate(wallet, predicate, amountToPredicate);
 
-      expect(toNumber(await predicate.getBalance())).toEqual(
-        initialPredicateBalance + amountToPredicate
-      );
+      expect(toNumber(await predicate.getBalance())).toBeGreaterThan(initialPredicateBalance);
```

Comment:
Mostly because of this: https://github.com/FuelLabs/fuels-ts/issues/2081

We have predicates being used by multiple test cases and since the tests are running concurrently, every type of verification that tries to predict a predicate balance will be flaky.

But `toBeGreaterThan` it is less flaky than `toEqual`